timeline_for_invoicing__add_zipcode/run.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+mkdir -p ${SCRIPT_DIR}/classes
+javac -classpath ${SCRIPT_DIR}/libs/jettison-1.5.4.jar \
```

Comment:
@marco-vit-pagopa possiamo includerlo nel nostro repo senza problemi di licenze?

Reply:
Mi sembra che la Apache License 2.0 sia piuttosto permissiva anche in tal senso.

Magari metto nella directory un file che indichi l'origine del jar.